spec/acceptance/01_zookeeper_spec.rb

```diff
@@ -18,16 +18,12 @@ class { 'java' :
         group  => 'zookeeper',
       }
 
-      $zookeeper_service_provider = $facts['os']['release']['major'] ? {
-        '6' => 'redhat',
-        '7' => 'systemd',
-      }
-
       class { 'zookeeper':
         install_method      => 'archive',
         archive_version     => '3.6.1',
-        service_provider    => $zookeeper_service_provider,
+        service_provider    => 'systemd',
```

Comment:
Looks like this was already supposed to work without passing it: https://github.com/deric/puppet-zookeeper/blob/4cf47591dde9f6efbde1635005c68cdf515b26cd/manifests/params.pp#L32-L44 so doesn't this also work:
```suggestion
```